lib/src/components/NxTabs/__tests__/NxStatefulTabs.test.tsx

```diff
@@ -5,29 +5,81 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { shallow } from 'enzyme';
-import 'jest-enzyme';
+
+import { render } from '@testing-library/react';
+import { rtlRender, rtlRenderElement, userEvent } from '../../../__testutils__/rtlUtils';
+
 import NxStatefulTabs from '../stateful/NxStatefulTabs';
+import { Props } from '../stateful/types';
 import NxTabList from '../NxTabList';
 import NxTab from '../NxTab';
 import NxTabPanel from '../NxTabPanel';
 
-describe('NxStatefulTabs', function () {
-  it('selects no tab when none is provided', function () {
-    const component = shallow(
-      <NxStatefulTabs>
-        <NxTabList>
-          <NxTab>Tab</NxTab>
-        </NxTabList>
-        <NxTabPanel>Tab Content</NxTabPanel>
-      </NxStatefulTabs>
-    );
+describe('NxStatefulTabs', function() {
+  const minimumProps = {
+    defaultActiveTab: -1,
+    children: (
+      <NxTabList>
+        <NxTab>Tabs 0</NxTab>
+      </NxTabList>
+    )
+  };
+
+  const quickRender = rtlRender<Props>(NxStatefulTabs, minimumProps);
+  const renderEl = rtlRenderElement<Props>(NxStatefulTabs, minimumProps);
+
+  it('merges any passed in className', function() {
+    const tabsWithAddedClassName = renderEl({ className: 'foo' });
+    const tabs = renderEl()!;
+
+    expect(tabsWithAddedClassName).toHaveClass('foo');
 
-    expect(component).toHaveProp('activeTab', undefined);
+    for (const cls of Array.from(tabs.classList)) {
+      expect(tabsWithAddedClassName).toHaveClass(cls);
+    }
   });
 
-  it('selects the default tab initially', function () {
-    const component = shallow(
+  it('selects no tab when none is provided', function() {
+    const { getByRole } = quickRender();
+    expect(getByRole('tab')).not.toHaveClass('active');
```

Comment:
`aria-selected`

Reply:
> `aria-selected`

Updated as part of: 6bacf16984a6f52b3dcc2726b315329746467423